airflow/providers/amazon/aws/utils/emailer.py

```diff
@@ -32,6 +32,7 @@ def send_email(
     mime_charset: str = 'utf-8',
     conn_id: str = 'aws_default',
     from_email: Optional[str] = None,
+    custom_headers: Optional[Dict[str, Any]] = None,
```

Comment:
Can the value be `Any`? Email headers should be in str, aren’t they?

Reply:
I used the same typings that were used in #19009 for consistency.  And it matches the signature of build_mime_message

Reply:
Let’s make this a separate discussion then.